### DISCUSSION TITLE
Trying build kernel for Poco X3 NFC (Surya)

### DISCUSSION THREAD

**Discussion by NiwiTheFox** [android/ndk#1812] [Q&A] [answered]
So far i was able to find answers on google until this point which where i am stuck rn

![image](https://user-images.githubusercontent.com/65979141/204660380-c6173596-4fa4-42da-9118-438acdaf9a89.png)

**Comment by NiwiTheFox** [comment]
what should i put into cross_compile_arm32 ?

**Comment by NiwiTheFox** [comment]
tried to do it this way but that just send me back where i started googling
![image](https://user-images.githubusercontent.com/65979141/204665366-ec145e4f-d045-40f2-bf3c-f7bbcc9cd78c.png)

**Comment by DanAlbert** [comment, marked as answer]
The NDK is for apps, not the kernel. Follow https://source.android.com/docs/setup/build/building-kernels for kernels.

**Reply by NiwiTheFox** [reply]
ooohhh
would make sense-
welp thanks
I will try that then
